_data/sw_clients.yml

```diff
@@ -441,6 +441,22 @@
           sasl-3.1:
         SASL:
           - plain
+    - name: Mutter
+      # maintainer: blipz
+      link: https://www.mutterirc.com/
+      os:
+        - ios
+      support:
+        stable:
+          cap-3.1:
+          away-notify:
+          batch:
+          multi-prefix:
+          sasl-3.1:
+          server-time:
+          userhost-in-names:
+      SASL:
```

Comment:
SASL should be indented two spaces in so it's under support, other than that looks great!